js/src/lib/Details/Aside.js

```diff
@@ -41,6 +43,10 @@ const Aside = (
         >
           {window.i18n.detail.try_in_runkit}
         </a>
+        {' \xB7 '}
```

Comment:
Good point, I can do that. Internally at FB we have a lint rule blocking all non-ASCII characters for some reason. I could directly embed it here though, you're right :D